### PR TITLE
pinentry styled and working

### PR DESCRIPTION
**Pull request by chrisnojima**

### CODE REVIEW

File: react-native/react/native/pinentry-listeners.desktop.js [hunk 1]
```diff
@@ -19,42 +19,83 @@ export default {
...
+    let pinentryWindow = new BrowserWindow({
+      width: 513, height: 230,
+      resizable: true,
       fullscreen: false,
-      show: false
+      show: false,
+      frame: false
     })
+
+    pinentryWindow.toggleDevTools()
```
Comment (cjb) [inline react-native/react/native/pinentry-listeners.desktop.js]: Should this be here?  Hard to tell whether it's conditionalized on dev etc.

**Comment by cjb** [comment]
LGTM % devtools!

A question for @malgorithms and design team (doesn't have to hold up this PR) -- pinentry currently has buttons for "Close" and "Cancel", where "Close" actually means "yes, here's my passphrase".  Should we maybe use a more descriptive verb?

File: react-native/react/native/pinentry-listeners.desktop.js (hunk 1 above)
Comment (chrisnojima) [inline react-native/react/native/pinentry-listeners.desktop.js]: Yup, i'll fix that up with the config flag, tnx!

File: react-native/react/common-adapters/header.desktop.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import React, {Component} from '../base-react'
+import commonStyles from '../styles/common'
+
+export default class Header extends Component {
+  render (): ReactElement {
```
Comment (MarcoPolo) [inline react-native/react/common-adapters/header.desktop.js]: There's an annotation here, but no flow annotation

**Comment by malgorithms** [comment]
@cjb - sure, "Submit" or "Ok" work for me generally; if you want to get really fancy, the pinentry could take a submit verb passed, and then whoever asks for the pinentry can set it to something appropriate such as "Login" or "Sign" or whatever.

**Pull request merged**